### ISSUE TITLE
Expected end date in Education Entry

### ISSUE THREAD

**Issue by andrefpoliveira** [sinaatalay/RenderCV#152]
In some cases, it can be helpful to have the expected end date instead of "present". It could show as, for example, `Sept. 2024 (Expected)`

**Comment by jw-shan** [comment]
You can implement this using [arbitrary keys](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/#using-arbitrary-keys) and [custom themes](https://docs.rendercv.com/user_guide/#creating-custom-themes-with-the-create-theme-command). Taking `engineeringresumes` as an example, first create a custom theme:
```
rendercv create-theme "mycustomtheme" --based-on "engineeringresumes"
```
Next, modify the file `./mycustomtheme/EducationEntry.j2.tex` as follows:
```
((* if not is_first_entry *))
\vspace{<<design.margins.entry_area.vertical_between>>}
((* endif *))

((* if entry.expected *))
\begin{twocolentry}{
    <<entry.date_string>> (expected)
}
((* else *))
((* if entry.date_string *))
\begin{twocolentry}{
    <<entry.date_string>>
}
((* else *))
\begin{onecolentry}
((* endif *))
((* endif *))
    \textbf{<<entry.institution>>}, ((* if entry.degree *))<<entry.degree>> in ((* endif *))<<entry.area>>((* if entry.location *)) -- <<entry.location>>((* endif *))
((* if entry.date_string *))
\end{twocolentry}
((* else *))
\end{onecolentry}
((* endif *))

((* for item in entry.highlights *))
    ((* if loop.first *))
\vspace{<<design.margins.highlights_area.top>>}
\begin{onecolentry}
    \begin{highlights}
    ((* endif *))
        \item <<item>>
    ((* if loop.last *))
    \end{highlights}
\end{onecolentry}
    ((* endif *))
((* endfor *))
```
In this example, lines 5–9 check whether there is an key called `expected` . If so, the date will appear with `(expected)` at the end. To create an education section, you would write:
```
    Education:
      - institution: xxx
        area: xxx
        degree: xxx
        date: 2024-06 
        expected: true
        highlights:
          - 'xxx'
```
This will display the date as `Jun 2024 (expected)` as you want. It would be appreciated if the author could incorporate this feature into the original code.

**Comment by tobybaratta** [comment]
FYI, if you don't put an `end_date`, it will (at least in the modern CV theme) default to `to present`, in the Education section. :)

**Comment by sinaatalay** [comment]
@tobybaratta Indeed, leaving `end_date` blank will always default to `present` in all themes.

Currently, we don't have a mechanism to write `end_date: 2024-12 (expected)` though. If anyone can think of how this would look in the YAML file, considering the [`locale_catalog` field](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/#locale_catalog-field), we may be able to do this.
